dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -136,50 +138,55 @@ private static void printStaticLinkageReport(StaticLinkageCheckReport report) {
   @VisibleForTesting
   static ImmutableList<Path> generateInputClasspathFromLinkageCheckOption(
       StaticLinkageCheckOption linkageCheckOption) throws RepositoryException {
-    ImmutableList.Builder<Path> jarFileBuilder = ImmutableList.builder();
+    List<Artifact> artifacts =
+        linkageCheckOption
+            .getArtifacts()
+            .stream()
+            .map(DefaultArtifact::new)
+            .collect(Collectors.toList());
 
-    // TODO(suztomo): add logic to convert Maven BOM to list of Maven coordinates as per README.md
-    jarFileBuilder.addAll(linkageCheckOption.getJarFiles());
-    for (String mavenCoordinates : linkageCheckOption.getArtifacts()) {
-      jarFileBuilder.addAll(coordinatesToClasspath(mavenCoordinates));
+    String bomCoordinates = linkageCheckOption.getBom();
```

Comment:
I think we need to refactor how we handle options. The types should be unified. See https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/255

Reply:
I'll merge the refactoring into this PR, once it goes to master.